lib/path.js

```diff
@@ -210,6 +221,9 @@ win32.join = function() {
 // to = 'C:\\orandea\\impl\\bbb'
 // The output of the function should be: '..\\..\\impl\\bbb'
 win32.relative = function(from, to) {
+  assertPath(from);
+  assertPath(to);
```

Comment:
This will be a little weird because the error will always be "path" no matter which is missing.